runhouse/resources/blobs/file.py

```diff
@@ -131,9 +131,12 @@ def to(
             return new_blob
 
         new_file = copy.copy(self)
-        new_file.local._folder = self._folder.to(
-            system=system, path=path, data_config=data_config
+        new_file.local._folder = folder(
+            path=str(Path(path).parent) if path is not None else path,
+            system=system,
+            data_config=data_config,
         )
+        new_file.write(self.fetch())
```

Comment:
@dongreenberg calling `self.fetch()` returns the file contents in byte form rather than the original raw text, what do you think is the best way to achieve copying over the file contents or even the file itself?